tests/unit-tests/IterableConfigTests.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  File.swift
+//  
+//
+//  Created by evan.greer@iterable.com on 4/5/23.
+//
+
+import XCTest
+
+final class IterableConfigTests: XCTestCase {
+    func testDefaultConfig() {
```

Comment:
Instead of testing the default config as a whole (which actually has defined defaults), I would just make a test instantiating a default API (using `initializeForTesting`) and then checking that it points to `IterableDataRegion.US`. And then another with the config pointing to EU, and check that the API keeps the EU value. Or wait to do the test until after the endpoints get integrated.

Reply:
I would remove it for now and add a new one when the endpoint changing code is in.